File: task_2/scatter.py
import pandas as pd
import matplotlib.pyplot as plt

# Charger le jeu de données
data = pd.read_csv('Housing.csv')

# Afficher les premières lignes du jeu de données pour vérifier le chargement
print(data.head())

# Graphique nuage de points (scatter plot) pour 'area' et 'price'
plt.figure(figsize=(8, 6))
plt.scatter(data["area"], data["price"], c="blue", alpha=0.7)
plt.xlabel("Superficie")
plt.ylabel("Prix")
plt.title("Superficie vs Prix")
plt.grid(True)
plt.show()

File: task_2/scatter.R
# Charger les données
df <- read.csv("Housing.csv")

# Afficher les premières lignes du dataset
head(df)
# Graphique de dispersion
plot(df$area, df$price, main="Graphique de dispersion de la superficie vs le prix", xlab="Superficie (en pieds carrés)", ylab="Prix (en dollars)", pch=19, col=rgb(0.2,0.4,0.6,0.5))
